src/components/CardModal/CardModal.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { screen, render, fireEvent } from '@testing-library/react';
+import { vi } from 'vitest';
+import { CardModal } from './CardModal';
+import { ICard } from '../../types/interfaces';
+import database from '../../Api/mockSource.json';
+
+describe('CardModal', () => {
+  const mockCard: ICard = database.results[0];
+
+  test('should render the card modal', () => {
+    const handleClose = vi.fn();
```

Comment:
if u use the same function in several places move her out of the describe and reuse in other places

Reply:
fixed